spec.bs

```diff
@@ -1734,14 +1742,18 @@ An auction config is a [=struct=] with the following items:
 : <dfn>decision logic url</dfn>
 :: A [=URL=].
   The URL to fetch the seller's JavaScript from.
-  <p class="note">The [=auction config/decision logic url=] [=origin=] will always be [=same origin=] with
-  [=auction config/seller=].</p>
+  <p class="note">
+    The [=auction config/decision logic url=] [=origin=] will always be [=same origin=] with
+    [=auction config/seller=].
+  </p>
 : <dfn>trusted scoring signals url</dfn>
 :: Null or a [=URL=].
   Provide a mechanism for making real-time data (information about a specific creative) available
   for use at scoring time, e.g. the results of some ad scanning system.
-  <p class="note">The [=auction config/trusted scoring signals url=] [=origin=] will always be [=same origin=] with
-  [=auction config/seller=].</p>
+  <p class="note">
+    The [=auction config/trusted scoring signals url=] [=origin=] will always be [=same origin=]
+    with [=auction config/seller=].
+  </p>
 : <dfn>interest group buyers</dfn>
 :: Null or a [=list=] of [=origin=].
   Owners of interest groups allowed to participate in the auction. Each [=origin's=] [=origin/scheme=]

```

Comment:
(several lines down -- Github doesn't let me put a comment there):

{{unsigned short}}s -> {{unsigned shorts}} should work, I think?